servers/audio_server.h

```diff
@@ -126,19 +126,52 @@ class AudioDriverManager {
 		MAX_DRIVERS = 10
 	};
 
+public:
+	enum DriverMode {
+		// User enables or disables audio, e.g. via button in editor
+		DRIVER_MODE_DISABLED = 1 << 0,
+		// Whether app is in focus
+		DRIVER_MODE_UNFOCUSSED = 1 << 1,
```

Comment:
```suggestion
		DRIVER_MODE_UNFOCUSED = 1 << 1,
```

Reply:
Ah yup indeed that is US (and more common) spelling it looks like! :grin: